21.1 mz dashboards eud (#135)
DevExpress/dotnet-eud: dashboard-for-web/articles/web-dashboard-designer-mode/data-analysis/dashboard-parameters/passing-parameter-values.md

> 

Reviewer: Line 7: In this topic, it describes how to pass the created dashboard parameter to the dashboard. For instance, you can include a dashboard parameter to a _WHERE_ clause of the SQL query or you can filter a dashboard dynamically according to the required parameter value(s).

Can we update this as follows?

`This topic describes how to pass the created dashboard parameter to the dashboard. For instance, you can include a dashboard parameter in the the SQL query's _WHERE_ clause or you can filter a dashboard dynamically according to the required parameter value(s).`

![wdd-configure-query-param-page2](../../../../images/img124955.png)

> To bind a dashboard parameter to an SQL query/stored procedure parameter in the [Dashboard Data Source Wizard](../../ui-elements/dialogs-and-wizards/dashboard-data-source-wizard.md), do the following:

Reviewer: ```suggestion
Do the following to bind a dashboard parameter to an SQL query or stored procedure parameter in the [Dashboard Data Source Wizard](../../ui-elements/dialogs-and-wizards/dashboard-data-source-wizard.md):
```

> * Select the existing query/stored procedure parameter or create a new query parameter using the **Add** button.

Reviewer: ```suggestion
* Select the existing query or stored procedure parameter, or use the **Add** button to create a new query parameter.
```

* Set the **Expression** as a parameter value and click the ellipsis button to invoke the Expression Editor for this parameter.

> * In the Expression Editor add the required dashboard parameter from the Parameters column.

Reviewer: ```suggestion
* In the Expression Editor, add the required dashboard parameter from the Parameters column.
```
